answerking_app/tests/test_categories.py

```diff
@@ -93,23 +103,27 @@ def test_post_valid_with_items_returns_ok(self):
         created_category: Category = Category.objects.filter(
             name="Vegetarian"
         )[0]
-        created_category_items: list[ItemType] = actual["items"]
         updated_list: QuerySet[Category] = Category.objects.all()
 
         # Assert
         self.assertNotIn(actual, old_list)
         self.assertIn(created_category, updated_list)
-        self.assertIn(item, created_category_items)
-        self.assertJSONResponse(expected, actual, response, 201)
+        self.assertCreateUpdateTime(expected, actual, response, 201)
```

Comment:
you are not asserting the rest of the JSONResponse

Reply:
same in other places

Reply:
it gets called in the custom methods assertUpdateTime and assertCreateUpdateTime

Reply:
oh yeah, my bad, missed that